src/components/sustainable/styledSustainable.js

```diff
@@ -0,0 +1,119 @@
+import styled from "styled-components/macro";
+import containerMonster from '../../images/container-monster.svg';
+
+export const SustainableWrapper = styled.section`
+  padding: 120px 0;
+  background: #F4F4F4;
+`;
+
+export const SustainableContent = styled.div`
+  /* 1340 width / 1440 width (1340 is from carousel button to button) */
+  width: 93.06%; 
+  margin: 0 auto;
+`;
+
+export const TitlesGroup = styled.div`
+  max-width: 598px;
+  margin: 0 auto;
+  padding-bottom: 60px;
+`;
+
+export const Title = styled.h2`
+  font-size: 44px;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0em;
+  text-align: center;
+  margin-bottom: 40px;
+`;
+
+export const Subtitle = styled.p`
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0em;
+  text-align: center;
+`;
+
+export const Carousel = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const Slide = styled.div`
+  width: 90.45%;       /* 1212 width / 1340 width */
+  height: 600px;
+  background: #fff;
+  box-shadow: 0px 2px 10px 0px #00000033;
+`;
+
+export const CarouselButton = styled.button`
+  width: 24px;
+  height: 24px;
+  border: none;
+  background: url(${props => props.image});
+  transition: transform .2s;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.33);
+  }
+`;
+
+export const BenefitsSection = styled.div`
+  width: 90.45%;       /* 1212 width / 1340 width */
+  height: 751px;       /* 871px (slide to next section) - 120px (padding) */
+  margin: 0 auto;
+  position: relative;
+`;
+
+export const LogisticsGroup = styled.div`
+  max-width: 286px;
+  display: flex;
+  gap: 37.5px;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  top: 110px;
+  left: 6.35%;         /* (191px -  114px) / 1212px */
+`;
```

Comment:
I know it's pixel perfect to make these have different gaps and widths according to the design, but it seems entirely unnecessary to me when we could just duplicate the same component 3 times.  If we ever get these images maybe we'll have a better idea.  For now I guess just leave it